### PR TITLE
obj Plugin Boto3 Migration (simplified for code review)

### PR DESCRIPTION
**Pull request by zliang-akamai**
## 📝 Description

Remove deprecated boto2 package and migrate to boto3.

This is the simplified PR of https://github.com/linode/linode-cli/pull/420

## ✔️ How to Test

`pytest tests/integration/test_obj_plugin.py`

Manually try out some obj commands

### CODE REVIEW

File: linodecli/plugins/obj/__init__.py [hunk 1]
```diff
@@ -215,131 +229,57 @@ def upload_object(get_client, args):  # pylint: disable=too-many-locals
...
-                    f"{file_path}, using {results[0]}"
-                )
-
-            file_path = results[0]
-
-        if not os.path.isfile(file_path):
-            print("No file {file_path}")
+        if platform.system() == "Windows" and "*" in f:
+            results = expand_globs(f)
+            files.extend(results)
```
[the remark this answers is not in the document]
Comment (zliang-akamai) [reply linodecli/plugins/obj/__init__.py:249]: Nice catch! Let me make a change accordingly.

**Review by jriddle-linode — APPROVED** [review]

**Comment by zliang-akamai** [comment]
/acctest sha=b7ef19be60c1da6a6d6a97730d3f82a769b767ae tests=test_obj_plugin.py

**Comment by zliang-akamai** [comment]
@lgarber-akamai acctest command doesn't work?

**Comment by zliang-akamai** [comment]
/acctest sha=2b4c9ccd02f67d23b8b76695a9b3d1a3289026e0 tests=test_obj_plugin.py

**Comment by lgarber-akamai** [comment]
@zliang-akamai Looks like `/acctest` is working, but it'll need to install `boto3` rather than `boto`

**Comment by zliang-akamai** [comment]
/acctest sha=f4be25156f5697784f279de73100c45e738898f7 tests=test_obj_plugin.py

**Review by lgarber-akamai — APPROVED** [review]
Unit and integration tests are passing locally. Nice work!

**Pull request merged**